v2/brigadier/src/jobs.ts

```diff
@@ -20,33 +61,177 @@ export class Job {
     this.event = event
   }
 
+  /**
+   * Runs the job.
+   * 
+   * When you run the job, Brigade runs all the containers, primary and sidecar.
+   * When the primary container exits, the job ends. If sidecars are still running
+   * at this point, Brigade terminates them after a short delay.
+   * 
+   * @returns A Promise which completes when the primary container completes. If the
+   * primary container succeeded (exited with code 0), the Promise resolves; if the
+   * primary container failed (exited with nonzero code) or timed out, the Promise
+   * rejects. Sidecars do not affect success or failure of the job.
+   */
   public run(): Promise<void> {
     return Promise.resolve()
   }
 
+  /**
+   * Gets the job logs.
+   * 
+   * If the job has multiple containers, this aggregates the logs from them all.
+   */
```

Comment:
This comment might apply in another spot or two that I failed to identify.

The description here is inconsistent with that the logic below does. Is it worth explaining in the comment that the logic here always resolves with the string "skipped logs," but that the Brigade worker "poly-fills" this at runtime with logic that communicates with the Brigade API?

Reply:
Yes, good point. I've proposed some wording for both this and the `run` function (which I think are the two affected). Would appreciate your feedback once it's pushed.